docs/tasks/exporting-and-importing.md

```diff
@@ -1,88 +1,187 @@
-# Exporting and Importing in Nuclio
+# Exporting and Importing Nuclio Functions and Projects
 
-This tutorial guides you through the process of exporting and importing functions and projects.
+This tutorial guides you through the process of using the Nuclio CLI (`nuctl`) to export and import Nuclio functions and projects.
 
 ### In this document
-- [Exporting a deployed function](#exporting-a-deployed-function)
-- [Importing a function](#importing-a-function)
-- [Redeploying an imported function](#redeploying-an-imported-function)
-- [Exporting a project](#exporting-a-project)
-- [Importing a project](#importing-a-project)
 
-## Exporting a deployed function
+- [Exporting deployed functions](#functions-export)
+- [Importing functions](#functions-import)
+- [Exporting projects](#projects-export)
+- [Importing projects](#projects-import)
+- [Deploying imported functions](#imported-functions-deploy)
 
-Once you have some functions deployed in nuclio (see [Deploying functions](/docs/tasks/deploying-functions.md)), you'll be able to export them for later import in this nuclio system or any other nuclio system. After which, it's recommended that you save the output to a file with a redirection (e.g. `command --with output > file.yaml` ).
+<a id="functions-export"></a>
+## Exporting deployed functions
 
-In order to export the function, run the command (remember to replace `function-name` with the relevant function name):
+You can  use the Nuclio CLI's `export functions` command (or the `export function` alias) to export the configurations of [deployed Nuclio functions](/docs/tasks/deploying-functions.md) in your environment ("export functions").
+You can save the exported configurations, for example, to a file, and [import](#functions-import) them later on any environment that is running Nuclio.
+
+To export a specific function, set the optional `<function>` argument to the name of the function to export, optionally followed by a colon (`:`) and the function's API version &mdash; i.e., `<function name>` or `<function name>:<function version>`:
 ```sh
-nuctl export function --namespace nuclio function-name
+nuctl export functions --namespace nuclio <function>
 ```
-The command will print the exported function config to the stdout.
-
-> **Note:** This command by default will export the function in yaml format. However, you can supply the flag `--output json` if you prefer a json output.
-
-## Importing a function
-
-Once you have [exported a function](#exporting-a-deployed-function), you can use the exported function config you saved in a file to import said function.
-
-Either run this command with the file path:
+For example:
+```sh
+nuctl export functions --namespace nuclio myfunction
+nuctl export functions --namespace nuclio myfunction:v1
```

Comment:
```suggestion
nuctl export functions --namespace nuclio myfunction:v1
```

why? not needed and I am not sure even supported (or a thing)

Reply:
@liranbg do you mean why we document the option of a function name with a version — `<function name>:version`? Because this is what I saw in the help: The original `export functions` usage string was `"functions [name[:version]]"` and I had confirmed this with Adam when I edited the doc. I have no problem removing this from the doc and help text, but please sync with @quaark  first.

Reply:
we dont support function versions, it should be omitted.

Reply:
@liranbg I removed the reference to exporting a specific function version both from the Markdown doc and from the help text — see commit 1dab774.